fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -1200,6 +1204,19 @@ protected ResponseFormatResolver buildResponseFormatResolver() {
         return new DefaultResponseFormatResolver();
     }
 
+    /**
+     * Returns the class to bind to {@link ResponseProcessorFactory}.
+     * <p>
+     * The ResponseProcessorFactory allows us to inject a custom {@link
+     * com.yahoo.bard.webservice.web.responseprocessors.ResponseProcessor} despite the fact that these processors depend
+     * on objects that are built uniquely for each request.
+     *
+     * @return A class that implements {@link ResponseProcessorFactory}.
+     */
+    protected Class<? extends ResponseProcessorFactory> buildResponseProcessorFactory() {
+        return ResultSetResponseProcessorFactory.class;
```

Comment:
Prob. not a blocker for this PR, but what if other `ResponseProcessor` implementations also need custom injection?

Reply:
Erm, that's the whole idea of this PR? If somebody wants to inject their
own custom `ResponseProcessor` then they:

1. Define their `CustomResponseProcessor`.
2. Define their `CustomResponseProcessorFactory`.
3. Override this method and return `CustomResponseProcessorFactory.class`

Reply:
Do you mean if they have non-empty constructors? Then they'd need to make their
constructor injectable so that HK2 can handle it.

Reply:
Make sense